### ISSUE TITLE
Option for caption outside of image

### ISSUE THREAD

**Issue by MaggiWuerze** [Zren/chrome-extension-imagus#4]
could you add an option to display the caption UNDER the image. This way it would not obscure the image when wrapped. I managed to do it via chrome dev console simply by setting the following properties of the surrounding div: 

- position: initial  <-- instead of absolute so the captions top sticks to the lower edge of the image
- overflow-y: auto; <-- so very long captions dont leave the screen and get a scrollbar if needed
- height: fit-content <--| this would need some tinkering or maybe an additional option. i ran with 100px.
- max-height:???px;  <--|

![image](https://user-images.githubusercontent.com/9729962/45422732-66711400-b691-11e8-919b-04c5e03ba089.png)

![image](https://user-images.githubusercontent.com/9729962/45423975-552a0680-b695-11e8-89f9-601ec2aaa0ea.png)

**Comment by DanielBiegler** [comment]
This I'd love as well. I can take a look at it later. Are you still interested in this @MaggiWuerze ?

**Comment by MaggiWuerze** [comment]
Yeah definitely, i can't live without this extension and this has bugged me for some time. Thanks for looking into it

**Comment by MaggiWuerze** [comment]
is there any progress regarding this feature?

**Comment by DanielBiegler** [comment]
Oh hey there, thanks for reminding me. I just spent some time to implement it here: https://github.com/DanielBiegler/chrome-extension-imagus/tree/feature/caption-under-media-with-wordwrap

I guess before this should be merged, the options page should be edited as well. I commited only twice, check it out.

Got to go now, will check it out later.

**Comment by MaggiWuerze** [comment]
really nice, looks good to me. 
Are you planning to give an option for max-height when you touch the options page??

**Comment by DanielBiegler** [comment]
Yeah one should also keep in mind the options for Captions in the imagus settings. There are options for the position. I hardcoded it to be on the bottom, it would be way better to respect the optionspage. I might check it out later but would be cool if you wanna help out.

**Comment by MaggiWuerze** [comment]
i'll have a look at how it could be positioned

**Comment by MaggiWuerze** [comment]
ok, it seems if you just switch the order of the `<img>` and the `<div>` containing the caption when the caption should be displayed on top and set the captions position to initial (as you already did) it works pretty flawless
